src/Tasks/dependency-check-build-task/task.json

```diff
@@ -83,6 +83,14 @@
       "required": false,
       "helpMarkDown": "Report output directory. On-prem build agents can specify a local directory to override the default location. The default location is the $COMMON_TESTRESULTSDIRECTORY\\dependency-check directory."
     },
+    {

```

Comment:
New option "Only warn for found violations"/"warnOnCVSSViolation".
If set the build step will succeed-with-issues instead of fail if a violation above (or equal to) the CVSS threshold was found.